### PR TITLE
{bio}[foss/2022a] pyBigWig v0.3.18 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3113.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/e978f6cca157394e23f73f191f1db2f0 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/3b9442f7205e5056aa63979da358a41a for a full test report.

File: easybuild/easyconfigs/p/pyBigWig/pyBigWig-0.3.18-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
```
Comment (verdurin) [inline easybuild/easyconfigs/p/pyBigWig/pyBigWig-0.3.18-foss-2022a.eb:1]: Also here, strictly speaking should be `PythonBundle`.

Comment (verdurin) [reply easybuild/easyconfigs/p/pyBigWig/pyBigWig-0.3.18-foss-2022a.eb:1]: Fine for now.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**